normandy/recipes/templates/admin/field_recipe_list.html

```diff
@@ -0,0 +1,9 @@
+{% if recipes %}
+  <ul>
+    {% for recipe in recipes %}
+      <li><a href="{% url 'admin:recipes_recipe_change' recipe.id %}">{{ recipe.name }}</a></li>
+    {% endfor %}
+  </ul>
+{% else %}
+  <p>Not used in any recipes.</p>
```

Comment:
And by "indiciate that" I mean it should show that the recipe is active or inactive with :white_check_mark: / :x:.